src/gui/virtualstudio.cpp

```diff
@@ -277,6 +280,9 @@ VirtualStudio::VirtualStudio(bool firstRun, QObject* parent)
     m_view.engine()->rootContext()->setContextProperty(
         QStringLiteral("updateChannelComboModel"),
         QVariant::fromValue(m_updateChannelOptions));
+    m_view.engine()->rootContext()->setContextProperty(
```

Comment:
The only relatively minor comment I have is this combo box could just be a check box so we can remove `m_feedbackDetectionOptions`

Reply:
I think we can probably just leave this in - primarily for the sake of uniformity on that settings page. If we really don't like it we can probably just change it later.